crud/common/utils.lua

```diff
@@ -120,9 +117,26 @@ local function tarantool_supports_fieldpaths()
     local patch = tonumber(major_minor_patch_parts[3])
 
     -- since Tarantool 2.3
-    __tarantool_supports_fieldpaths = major >= 2 and (minor > 3 or minor == 3 and patch >= 1)
+    features.tarantool_supports_fieldpaths = major >= 2 and (minor > 3 or minor == 3 and patch >= 1)
+
+    -- since Tarantool 2.4
+    features.tarantool_supports_uuids = major >= 2 and (minor > 4 or minor == 4 and patch >= 1)
+end
+
+local function tarantool_supports_fieldpaths()
+    if features.tarantool_supports_fieldpaths == nil then
+        determine_enabled_features()
+    end
+
+    return features.tarantool_supports_fieldpaths
+end
+
+function utils.tarantool_supports_uuids()
+    if features.tarantool_supports_uuids == nil then
+        determine_enabled_features()
+    end
 
-    return __tarantool_supports_fieldpaths
+    return features.tarantool_supports_uuids
```

Comment:
I like this approach, but what about naming?

For example,
`enabled_tarantool_features.fieldpaths`
`enabled_tarantool_features.uuids`

Reply:
We can't call it just "uuids" since exact feature name "indexes by uuid" (https://github.com/tarantool/tarantool/commit/b238def8065d20070dcdc50b54c2536f1de4c7c7)

Reply:
Renamed